docs/specification/May2019Ballot.md

```diff
@@ -538,10 +561,49 @@ Context | Optionality | FHIR operation to generate context|  Description
 #### Example: 
 ```
 {
- }
+  "context": [
+    {
+      "key": "patient",
+      "resource": {
+        "resourceType": "Patient",
+        "id": "ewUbXT9RWEbSj5wPEdgRaBw3",
+        "identifier": [
+          {
+            "system": "urn:oid:1.2.840.114350",
+            "value": "185444"
+          },
+          {
+            "system": "urn:oid:1.2.840.114350.1.13.861.1.7.5.737384.27000",
+            "value": "2667"
+          }
+        ]
+      }
+    },
+    {
+      "key": "study",
+      "resource": {
+        "resourceType": "ImagingStudy",
+        "id": "8i7tbu6fby5ftfbku6fniuf",
+        "uid": "urn:oid:2.16.124.113543.6003.1154777499.30246.19789.3503430045",
+        "identifier": [
+          {
+            "system": "7678",
```

Comment:
Similar comment as above, except I think this is syntactically invalid. This I would expect to probably be an oid system like above? Not sure there's a good "type" that would apply here.

Reply:
I don't know enough about the FHIR identifiers to determine what should go here. You have any ideas?